charts/openproject/templates/web-deployment.yaml

```diff
@@ -1,3 +1,5 @@
+{{- $image_tag := .Values.image.tag | required ".Values.image.tag is required. Please use the -slim variant from OpenProject and fix it to a full release version X.Y.Z" -}}
```

Comment:
Perhaps a link to https://hub.docker.com/r/openproject/openproject/tags?name=. - or our github releases page or our website would be good here so people can find the necessary version easily?